packages/core/src/attestation/Attestation.chain.ts

```diff
@@ -21,7 +21,7 @@ import { ConfigService } from '@kiltprotocol/config'
 import { BlockchainApiConnection } from '@kiltprotocol/chain-helpers'
 import { DidUtils } from '@kiltprotocol/did'
 import { BN } from '@polkadot/util'
-import { Attestation } from './Attestation'
+import { Attestation } from './Attestation.js'
 import type { DelegationNodeId } from '../delegation/DelegationDecoder'
```

Comment:
Since `type` imports are completely removed on transpilation (and we only need the extensions for the es modules), I didn't add them here.
Also eslint doesn't warn about them, so harder to catch